packages/video_player/video_player/test/video_player_test.dart

```diff
@@ -657,6 +657,11 @@ class FakeVideoPlayerPlatform extends VideoPlayerApiTest {
     calls.add('setVolume');
   }
 
+  @override
+  void setPlaybackSpeed(PlaybackSpeedMessage arg) {
+    // todo: implement as part of completing https://github.com/flutter/plugins/pull/3031
```

Comment:
I have already implemented this in the said PR, but I only want to include the tests in part 3/3 of that PR.  

That was the original plan anyway 😅